Pod/Classes/URBNAlertAction.m

```diff
@@ -25,7 +26,14 @@ + (URBNAlertAction *)actionWithTitle:(NSString *)title actionType:(URBNAlertActi
 }
 
 - (BOOL)isButton {
-    return (self.actionType == URBNAlertActionTypeNormal || self.actionType == URBNAlertActionTypeDestructive);
+    return (self.actionType == URBNAlertActionTypeNormal || self.actionType == URBNAlertActionTypeDestructive || self.actionType == URBNAlertActionTypeCancel);
+}
+
+- (void)setEnabled:(BOOL)enabled {
+    if (self.isButton && self.actionButton) {
+        [self.actionButton setEnabled:enabled];
+        self.actionButton.alpha = enabled ? 1.f : 0.5f;
```

Comment:
This will do for now, but I would like you to think about disabling a button actually means from a styling perspective. I think more than likely we would want to be able to supply different styles for different button states.